wenet/transformer/ctc.py

```diff
@@ -67,3 +68,63 @@ def argmax(self, hs_pad: torch.Tensor) -> torch.Tensor:
             torch.Tensor: argmax applied 2d tensor (B, Tmax)
         """
         return torch.argmax(self.ctc_lo(hs_pad), dim=2)
+
+
+    def forced_align(self, ctc_probs: torch.Tensor, y: torch.Tensor,
+                     blank_id=0) -> list:
```

Comment:
why add this function in CTC module, it can be a standalone function since it is not related to CTC module.